csun-metalab/nomi
# NAM-557 -- UX: Change term labels away from next, previous current, to the actual term

# Description
Updated term selector to be `Season 'year` .

# Testing
1. Login as `steve` .
2. Check to see the `term-selector` labels display the `Season` followed by `Year Suffix` .
2. Ensure the term selectors are pleasing to look at; no errors appear in console; responsive.
